CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 0.21.0-rc.0 / 2020-06-02
+
+* [CHANGE] [amtool] Remove default assignment of environment variables. #2161
+* [CHANGE] Enforce 512KB event size limit for Pagerduty events. #2225
+* [ENHANCEMENT] [Slack] Add channel when logging errors. #2177
+* [ENHANCEMENT] [Cluster] Log error on reconnect failures. #2260
```

Comment:
I'd combine all the logging stuff into one entry. We'd usually not even mention logging stuff.